fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfAttachment.java

```diff
@@ -136,6 +136,10 @@ public static String generate(CadfAttachment obj)
 
         public static void generate(CadfAttachment obj, JsonGenerator generator)
                 throws IOException {
+            // If the CADF attachment is null, then simply skip it.
+            if (obj == null) {
```

Comment:
Maybe you can use `ObjectUtils.isEmpty` to validade this obj.
Document [link](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ObjectUtils.html)